packages/router/src/bindings/messaging/auctionRequest.ts

```diff
@@ -41,9 +42,19 @@ export const auctionRequestBinding = async (
     logger.error("No data in auction request", requestContext, methodContext, err);
     return;
   }
+
   // On every new auction broadcast, route to the new auction handler
   logger.info("Received auction request", requestContext, methodContext);
   const { bid, bidSignature } = await newAuction(data, requestContext);
+
+  // check if new auction happens in 5s or not
+  const lastAttemptTime = lastAuctionMap.get(`${bid.user}-${bid.router}`) as number;
+  const currentTime = await getNtpTimeSeconds();
+  if (lastAttemptTime && lastAttemptTime + AUCTION_REQUEST_LIMIT > currentTime) {
```

Comment:
I think this code belongs in `operations/auction.ts` since it's a part of the auction business logic.